courses/machine_learning/deepdive/09_sequence_keras/translate/translate_model.ipynb

```diff
@@ -0,0 +1,809 @@
+{
```

Comment:
<ul><li>formatting (pep 8):</li></ul><p><br></p><p>URL = 'http://storage.googleapis.com/download.tensorflow.org/data/spa-eng.zip'</p><p><br></p><p>path_to_zip = tf.keras.utils.get_file(</p><p>&nbsp;&nbsp;'spa-eng.zip',</p><p>&nbsp;&nbsp;origin=URL,</p><p>&nbsp;&nbsp;extract=True</p><p>&nbsp;&nbsp;)</p><p><br></p><p>path_to_file = os.path.dirname(path_to_zip) + "/spa-eng/spa.txt"</p><p><br></p><ul><li>also: os.path.join would be better than "path1" + "path2" arithmetic, avoiding the problems with "/" </li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>